tsconfig.json

```diff
@@ -15,7 +15,6 @@
 		"esModuleInterop": true,
 		"skipLibCheck": false,
 		"module": "esnext",
-		"typeRoots": [ "**/node_modules/@types" ],
```

Comment:
This should be able to stay removed. tsconfig by default loads types from `@types` and some other places so doing this narrows what it will pick up.

Reply:
I added a `types` folder in root for type definitions as well, I am curious if it will pick that up automatically.

Reply:
@louwie17

From the docs "This config file will include all packages under ./typings and ./vendor/types, and no packages from ./node_modules/@types. All paths are relative to the tsconfig.json."

So perhaps not. Can I suggest you rename your folder to one of the supported folder names? typings seems good.

Reply:
Sure that would be great, the folder is called `typings` I guess this doesn't get automatically picked up either (as from your previous comment)?

Reply:
@louwie17 sounds good, I'll look at changing this to support that. Thanks for raising that!

Reply:
I've reverted my changes here and opted to manually declare everything we want to include.